### DISCUSSION TITLE
Ho to make Action button to become inline with search and filter action ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5264] [Help]
Hi All,

I'm using https://filamentphp.com/plugins/alperenersoy-export.
How to move the export button to become inline with search and filter action ? So I no need extra line just for export button.

![image](https://user-images.githubusercontent.com/43429317/207911107-cd0ca371-780f-4bd6-ade1-dd24d2647888.png)

**Comment by ManojKiranA** [comment]
It's not possible ATM I think. But you can override views to achieve this

**Reply by altimusin** [reply]
So.. What if,  I move Export button to list page.
Is it possible or not ?,  because the package only have **2** kinds of action.

**1. bulkActions**
```
$table->bulkActions([ 
    ...
    FilamentExportBulkAction::make('export')
    ...
]);
```
**2. headerActions**
```
 $table->headerActions([  
      ...
      FilamentExportHeaderAction::make('export')
      ...
  ]);    
```

![image](https://user-images.githubusercontent.com/43429317/208056692-8fcc3e9d-8f42-44f9-ba5c-9b33b7171d43.png)


I got error when tried this code.
```
<?php

namespace App\Filament\Resources\EmployeeResource\Pages;

use AlperenErsoy\FilamentExport\Actions\FilamentExportHeaderAction;
use App\Filament\Resources\EmployeeResource;
use Filament\Pages\Actions;
use Filament\Resources\Pages\ListRecords;

class ListEmployees extends ListRecords
{
    protected static string $resource = EmployeeResource::class;

    protected function getActions(): array
    {
        return [
            Actions\CreateAction::make(),
            FilamentExportHeaderAction::make('export')->button()->disablePreview()->disableAdditionalColumns()
        ];
    }
}
```

![image](https://user-images.githubusercontent.com/43429317/208059382-62f10d97-f6c8-48a6-9761-ac1c31eedb3d.png)

**Reply by ManojKiranA** [reply]
I think you can only use the `FilamentExportHeaderAction` class in table Header Actions I think.

**Reply by ArtMin96** [reply]
@altimusin I think for this error you can use `Filament\Forms\Concerns\BelongsToLivewire` trait in your class

**Comment by R70YNS** [comment]
Excellent way of doing this [here](https://filamentexamples.com/tutorial/move-create-button-to-search-and-filters-toolbar)
